README.md

```diff
@@ -1,2 +1,32 @@
-# HelloMicroverse1
-Hello Microverse Task
+
+# Project Name: Hello Microverse
+
+> Hello Microverse Task 
+
```

Comment:
- Please make your PR summary more descriptive, as it currently only contains `Add linters in github/workflows` 

**Hint**

- Create a new branch according to the GitHub flow rules.
- Added a .gitignore file.
- Set up a linter for HTML and CSS and made sure that any files that are not relevant are ignored by git.
- Added a descriptive README file to your project .
- Add 3 files (each one in a separate commit):
- Added a.html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
- Added a .css file that includes one class with styles for your header (make it in your favorite color).
- A test.md file with the text "This file should be ignored by git".

Reply:
Thank you so much... would you suggest that i add the content you've sent in your review comment ?